crates/solver/src/driver.rs

```diff
@@ -182,8 +182,8 @@ impl Driver {
             Ok(receipt) => {
                 let name = solver.name();
                 tracing::info!(
-                    "Successfully submitted {} settlement: {:?}",
-                    name,
+                    "Successfully submitted settlement id {} as tx hash {}",
```

Comment:
I found it is generally more useful to use `{:?}` for transaction hash because of the hash length, since with `{}` only part of the hash is logged making it useless...

Reply:
Fixed. Didn't intentionally change this.